docs/section10/10-4.md

```diff
@@ -0,0 +1,73 @@
+# ASHVAC – Rule 10-4  
+
+**Rule ID:** 10-4  
+ 
+**Rule Description:** Baseline HVAC systems using fossil fuel shall be modeled using natural gas. Except where natural gas is not available for the proposed building site, propane shall be used as the heating fuel.  
+
+**Rule Assertion:** B-RMR = expected value                                           
+**Appendix G Section:** Section G3.1-10 HVAC Systems for the baseline building   
+**Appendix G Section Reference:**   
+**Data Lookup:** None   
+**Evaluation Context:** Evaluate each HeatingVentilationAirAconditioningSystem   
+
+**Applicability Checks:**   None 
+
+**Function Call:**  
+
+1. get_baseline_system_types()
+2. get_fuels_modled_in_RMR()  
+
```

Comment:
should we have an applicability check here to see if baseline system is of type where heating fuel should be a fossil fuel? else this rule is NA, correct?

Reply:
I read the previous comments and realized that we had discussed this already and decided against it.

Reply:
Ok, comment closed.